livy/src/main/java/org/apache/zeppelin/livy/LivySparkSQLInterpreter.java

```diff
@@ -197,7 +200,18 @@ public FormType getFormType() {
     return rows;
   }
 
-  protected List<String> parseSQLOutput(String output) {
+  protected List<String> parseSQLOutput(String str) {
+    String fullWidthRegex = "([" +
+            "\u1100-\u115F" +
+            "\u2E80-\uA4CF" +
+            "\uAC00-\uD7A3" +
+            "\uF900-\uFAFF" +
+            "\uFE10-\uFE19" +
+            "\uFE30-\uFE6F" +
+            "\uFF00-\uFF60" +
+            "\uFFE0-\uFFE6" +
+            "])";
+    String output = str.replaceAll(fullWidthRegex, "$1\u0001");
```

Comment:
Use reg expression doesn't seems an elegant approach to me, could you find thrid party library that has such ability ?

Reply:
sorry,  I have no idea neither, this regx is copied from spark source.

Reply:
Could you link where these spark source code ?

Reply:
org.apache.spark.util.Utils#fullWidthRegex
![image](https://user-images.githubusercontent.com/47968604/126096656-3c21bd6a-5011-4b92-8b89-1d9f45c47ba0.png)
this picture above is code where the chinese character has two placeholders

Reply:
Thanks @zhxiaoping Could you add more comment ? Especially please comment where the spark code you refer ?

Reply:
the regex is refered to org.apache.spark.util.Utils#fullWidthRegex

![image](https://user-images.githubusercontent.com/47968604/126423231-b7586739-1f5c-4529-8403-cc6115dee2ba.png)

for spark chinese character has two placeholder, one placeholder is one char.
for zeppelin chinese has only one placeholder.  
they have different standards.
so zeppelin can not parse columns based on column size.
just because zeppelin take chinese character as one placeholder, but actually it is two placeholder.

this pr do two things
the first one thing  insert a special character (/u0001)  which nerver use  after every chinese character,
so zeppline can split string correctly,  replace /u0001 with empty string, before add record to rows

the second thing  avoid that chinese character is escaped.

Reply:
I mean to add comment in your PR so that the next contributor can understand what this change is for

Reply:
added comment in code

Reply:
Thanks @zhxiaoping Let's wait for the CI